profile_collection/startup/00-startup.py

```diff
@@ -370,7 +372,12 @@ def insert(self, name, doc):
 from nslsii import configure_base, configure_olog
 
 # configure_base(get_ipython().user_ns, db_new, bec=False)
-configure_base(get_ipython().user_ns, db, bec=False)
+# We have an error:
+# UserWarning: Couldn't start log: Log file is already active: 
+# /home/xf03id/logs/ophyd_user_log_xf03id.log
```

Comment:
@jklynch, this is the IPython logging issue we've observed today.

Reply:
Maybe it is this line from ipython_ophyd/profile_collection/ipython_config.py:

```python
# Start logging to the given file in append mode.
c.TerminalInteractiveShell.logappend = '/home/xf03id/logs/ophyd_user_log_xf03id.log'
```

Reply:
You could specify that file with BLUESKY_IPYTHON_LOG_FILE.

Reply:
Indeed! Not sure why I wasn't able to find it with `git grep`... probably I was in the `startup/` directory that time.

That's the place in the code:
https://github.com/NSLS-II-HXN/ipython_ophyd/blob/d59ecd4799f4d212525aca1e1956cc989405bb48/profile_collection/ipython_config.py#L297-L298